pbsmetrics/metrics.go

```diff
@@ -146,6 +146,7 @@ const (
 	AdapterErrorBadInput          AdapterError = "badinput"
 	AdapterErrorBadServerResponse AdapterError = "badserverresponse"
 	AdapterErrorTimeout           AdapterError = "timeout"
+	AdapterErrorFailedToBid       AdapterError = "failedtobid"
```

Comment:
Name seems a bit confusing to me... if you were unfamiliar with the project, how do you think this would compare with `nobid`?

Maybe `failedtorequest` would be more accurate?

Reply:
Yes, that is what I use elsewhere, Not sure how I got off track here. Thanks